packages/fxa-auth-server/test/local/routes/utils/clients.js

```diff
@@ -108,7 +108,7 @@ describe('clientUtils.formatLocation', () => {
     assert.equal(log.warn.callCount, 0);
   });
 
-  it('unsets location if it fails', () => {
+  it.skip('unsets location if it fails', () => {
```

Comment:
Skippy skip?

Reply:
Can we discuss this test? In general I am not really sure I understand the intent here. This test passed previously cause something like ‘wibble’ would result in request.app.locale being undefined. Now that we use determineLocale, which relies on parseAcceptLanguage to determine the locale, we always make a best effort to return something. In the event the acceptLanguage is nonsense, the returned value will now be the default which is ‘en’.

At any rate, I am not sure why having a bogus language should result in the location info being discounted... I do understand that without a proper locale, the location text can’t be properly translated, however, simply wiping it out seems odd. Isn’t this kind of like saying if you can’t determine the language render no text at all? Isn’t something better than nothing?

Reply:
Hmm yeah I'm also not sure why we would want to just throw out the entire thing if something funky happens when trying to determine the locale. It makes sense to me to just default to `'en'` since that's the language we're going to show the user if it can't be determined. 

Maybe the idea was that if that's invalid, the rest of it is probably also invalid in some way, so throw out everything. I'm personally fine with ripping the test out, and showing the location info regardless. We also have other tests around bogus location info (we send stuff through `safeUserAgent`, in at least emails, not entirely sure on "connected services") so I feel like we're covered. I'll tag @vbudhram here in case there's more context he may have but if we all think it should be ripped out then 🤷‍♀️

Reply:
It seems like the code for formatting location is pretty well covered, and even verifies the location remains untranslaged when the language is english. So I am thinking delete it. We should now be covered by the en case.